recipe/activate.sh

```diff
@@ -1,7 +1,31 @@
 #!/bin/bash
 
-if [[ -n "$CXX" ]]; then
-    export NVCC_PREPEND_FLAGS_BACKUP="${NVCC_PREPEND_FLAGS}"
-    export NVCC_PREPEND_FLAGS="${NVCC_PREPEND_FLAGS} -ccbin=${CXX}"
+export CFLAGS_BACKUP="${CFLAGS}"
+export CPPFLAGS_BACKUP="${CPPFLAGS}"
+export CXXFLAGS_BACKUP="${CXXFLAGS}"
+export NVCC_PREPEND_FLAGS_BACKUP="${NVCC_PREPEND_FLAGS}"
+
+if [ "${CONDA_BUILD:-0}" = "1" ]; then
+    if [ -z "${CXX+x}" ]; then echo 'cuda-nvcc: Please add the `compiler("c")` and `compiler("cxx")` packages to the environment.'; exit 1; fi
```

Comment:
Would prefer multiple lines for readability.
```suggestion
    if [ -z "${CXX+x}" ]; then
        echo 'cuda-nvcc: Please add the `compiler("c")` and `compiler("cxx")` packages to the environment.'
        exit 1
    fi
```